.github/workflows/vcpkg_docker.yml

```diff
@@ -0,0 +1,57 @@
+name: Docker Build Images
+
+on:
+  schedule:
+    # Once every Wednesday at 00:00
+    - cron: '0 0 * * 3'
+  push:
+    # branches:
+    #   - master
+    paths:
+      - 'docker/**'
+      - '.github/workflows/vcpkg_docker.yml'
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - { distro: 'ubuntu', version: '18.04' }
+          - { distro: 'ubuntu', version: '20.04' }
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build and Push
+        working-directory: docker
+        run: |
+          docker build -f Dockerfile.ubuntu.vcpkg --target caching --no-cache --build-arg "DISTRO_VERSION=${{ matrix.container.version }}" -t "${DOCKER_TAG}" .
+          docker login docker.pkg.github.com -u publisher -p "${GITHUB_PACKAGE_REGISTRY_TOKEN}"
+          for i in 1 2 3; do docker push "${DOCKER_TAG}" && break || sleep 10; done
+          docker logout docker.pkg.github.com
+        env:
+          DOCKER_TAG: docker.pkg.github.com/trailofbits/cxx-common/vcpkg-builder-${{ matrix.container.distro }}:${{ matrix.container.version }}
+          GITHUB_PACKAGE_REGISTRY_TOKEN: ${{  secrets.GITHUB_PACKAGE_REGISTRY_TOKEN  }}
+
+      # - name: Set up Docker Buildx
+      #   uses: docker/setup-buildx-action@v1.0.3
+
+      # - name: Login to GitHub Container Registry
+      #   uses: docker/login-action@v1
+      #   with:
+      #     registry: docker.pkg.github.com
+      #     username: ${{ github.repository_owner }}
+      #     password: ${{ secrets.GITHUB_TOKEN }}
+
+      # - name: Build and push
+      #   id: docker_build
+      #   uses: docker/build-push-action@v2.2.1
+      #   with:
+      #     file: ./docker/Dockerfile.${{ matrix.container.distro }}.vcpkg
+      #     target: caching
+      #     context: ./docker
+      #     push: true
+      #     tags: docker.pkg.github.com/trailofbits/cxx-common/vcpkg-builder-${{ matrix.container.distro }}:${{ matrix.container.version }}
+      #     build-args: DISTRO_VERSION=${{ matrix.container.version }}
```

Comment:
For some reason using the official Docker Actions doesn't work with the `docker.pkg.github.com` registry